src/ethtool/Makefile

```diff
@@ -11,6 +11,10 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	git clone https://salsa.debian.org/kernel-team/ethtool/
 	pushd ethtool
 	git checkout tags/debian/1%$(ETHTOOL_VERSION_BASE)-1
+ifneq (,(filter $(CONFIGURED_ARCH),arm64 armhf))
```

Comment:
>CONFIGURED_ARCH),arm64 armhf)) [](start = 18, length = 30)

I am not an expert on ethtool. Seems debian official source package could be built without any change. Could you investigate why?
[https://packages.debian.org/bullseye/ethtool](https://packages.debian.org/bullseye/ethtool)

Reply:
why do we configure ethtool differently on arm64 and armhf?

Reply:
Hi @lguohan ,

I've updated the description about this.